CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 ## 2.2.0 (Unreleased)
+
+DOCS:
+* Add docs for common source properties ([#50](https://github.com/terraform-providers/terraform-provider-sumologic/pull/50))
```

Comment:
On the Changelog we edit we write (GH-50), we don't put the link
The hashicorp bot will go and replace these